epub33/explainers/EPUB33-explainer.md

```diff
@@ -0,0 +1,134 @@
+
+EPUB 3.3 includes the following specifications:
+
+* [EPUB 3.3 Overview](https://w3c.github.io/epub-specs/epub33/overview/), a non-normative overview of EPUB 3.3
+* [EPUB 3.3 Core](https://w3c.github.io/epub-specs/epub33/core/), the specification of the file format
+* [EPUB 3.3 Reading Systems](https://w3c.github.io/epub-specs/epub33/rs/), the specification for user agents, known as EPUB Reading Systems. 
+* [EPUB Accessibility 1.1](https://w3c.github.io/epub-specs/epub33/a11y/)
+* [EPUB Accessibility Techniques 1.1](https://w3c.github.io/epub-specs/epub33/a11y-tech/)
+
+## Goals
+
+## Non-goals
+
+
+## Security and Privacy Questionnaire
+
+EPUB files are presented to end users by EPUB Reading Systems, which can be constructed as web applications but are more likely to be native apps. The EPUB specifications define the file format in great detail, but generally give user agents wide latitude, subject mostly to the requirements of the embedded technologies, such as HTML, SVG, XML, and CSS. 
+
+This makes it hard to think about security issues. In order to read EPUB files, I generally have to give personally-identifying information like a credit card number to the ebook retailer. They have likely altered the original EPUB, for example by applying digital rights management to the constituent files. Most of the security implications around EPUB depend on the architectural details and the business model of the reading system, rather than the specifications themselves. 
+
+It's hard to even reason about how HTML’s security issues affect EPUB. If I read an EPUB on the Google Play app on an Android tablet, what risks am I exposed to? Can any of those risks be mitigated by the design of the EPUB specification?
+
+
+
+### 1. What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
+
+EPUB Reading Systems typically collect information on the reading habits of their users. In some cases this information is necessary to provide a good user experience; in other cases the information is collected for other users—the reading system company or the publisher of the book. 
+
+1. Tracking reading position. You really do want the reading system to remember that you just started chapter 5. 
+2. Remembering highlights, notes, bookmarks. 
+3. Remembering user preferences around fonts, visual themes (including dark/light), text justification, etc. 
+4. Recording progress through a book and whether a book was finished. 
+5. Some reading systems track every user reading session, including the time of day, the duration, how many pages were read, what book, the user’s IP address if a web-based reader, etc. 
+
```

Comment:
Worth repeating what I said above: there is an exchange of info in some sense, but only for the same publication.

Reply:
Moot as this has been deleted from the explainer (and remains in the security questionnaire).